spec/git_reflow/all-approval/git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
```

Comment:
Yes, I moved this git_reflow_spec.rb test into `all-approval` folder because I wanted to create 2 separate spec workflows to test each git configuration. There are 2 git_reflow_spec tests one in `all-approval` and one in `2-lgtm`. This way we can test the complete workflow (from correct git commands running, to the correct output, merge success, failure, etc.) for each configuration. Yes, many of these tests are duplicated across both folders, but I see value as comprehensive tests should test each of these functionalities for each git config to ensure that both work as expected. IE: Each branch or use case should be tested separately. Also, more tests never hurt :-)